lib/ansible/plugins/inventory/constructed.py

```diff
@@ -63,10 +63,10 @@
 from ansible import constants as C
 from ansible.errors import AnsibleParserError
 from ansible.inventory.helpers import get_group_vars
-from ansible.plugins.cache import FactCache
 from ansible.plugins.inventory import BaseInventoryPlugin, Constructable
 from ansible.module_utils._text import to_native
 from ansible.utils.vars import combine_vars
+from ansible.vars.fact_cache import FactCache
```

Comment:
this should not be used, it should be querying vars manager instead, we updated other plugins the same way, but have not had gotten around to this one yet.

Reply:
Removed in https://github.com/ansible/ansible/pull/49516/commits/5beae97802cd2a14bbbbb8aa0174143170ff2888

Reply:
We talked this over on slack and decided to remove FactCache from constructed separately because constructed should be using vars manager instead of FactsCache but there's some problem with doing that at the moment.